src/client/services/gitHubService.ts

```diff
@@ -0,0 +1,26 @@
+import GitHub from 'github-api';
+
+export interface IGithubService {
+  /**
+   * Returns the gist of the latest commit for the given owner/repo.
+   */
+  getRepoLastCommitGist(owner: string, repo: string): Promise<{ message: string }>;
+}
+
+export class GitHubService implements IGithubService {
+  constructor(private gitHubApi: GitHub) {
+    console.log(this.gitHubApi);
+  }
+
+  public async getRepoLastCommitGist(owner: string, repo: string): Promise<{ message: string }> {
```

Comment:
Why not just return a string?

Reply:
I made it extendable from the thought that in the future  we might want extra data for the last commit  gist e.g when was it made ?  (so we could display time since),  who made it, a link to the commit and such

Reply:
If you think the extra data will never  be needed then Ill make it a simple string